Mlem/Views/Shared/Composer/PostComposerView.swift

```diff
@@ -103,17 +92,32 @@ struct PostComposerView: View {
                             }
                             .padding(.top)
                             .padding(.horizontal)
+                            .onChange(of: postTitle) { newValue in
+                                do {
+                                    if let regex = siteInformation.slurFilterRegex {
+                                        if let output = try regex.firstMatch(in: newValue.lowercased()) {
+                                            titleSlurMatch = String(newValue[output.range])
+                                        } else {
+                                            titleSlurMatch = nil
+                                        }
+                                    }
+                                } catch {
+                                    print("REGEX FAILED")
+                                }
```

Comment:
*nit*

This logic is reused for the body and comments so it probably makes sense to extract it to `func slurCheck(_ toCheck: String) -> String?` in `SiteInformationTracker` and just have this do `titleSlurMatch = siteInformation.slurCheck(newValue)`

Reply:
Sure, good idea 👍